src/org/sosy_lab/common/log/LoggingOptions.java

```diff
@@ -92,6 +92,8 @@ public LoggingOptions(Configuration config) throws InvalidConfigurationException
     config.inject(this);
   }
 
+  public LoggingOptions() {}
```

Comment:
This needs to be `protected`, not `public`. Probably should also get a comment that explains that it is for inheritance.